### PR TITLE
chore: add node v18 to test workflow

### PR DESCRIPTION
**Pull request by amareshsm**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Add node v18 to test workflow.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 17.x, 18.x]
```
Comment (alexander-akait) [inline .github/workflows/nodejs.yml:55]: Can you add only 18, we don't need `17` here, it is not LTS

Comment (amareshsm) [reply .github/workflows/nodejs.yml]: done

**Comment by ricardogobbosouza** [comment]
unfortunately it's crashing in v18

**Comment by ricardogobbosouza** [comment]
#284

**Pull request closed without merging**